### PR TITLE
Migrate to Heroku review apps

### PR DESCRIPTION
**Pull request by taktran**
From https://github.com/Financial-Times/next/issues/250

### CODE REVIEW

**Comment by taktran** [comment]
May need some more work, as this is not working yet: https://github.com/Financial-Times/next-syndication-dl/pull/7

**Comment by taktran** [comment]
This is ready for review.

Fixed the issues with https://github.com/Financial-Times/next-syndication-dl/pull/7, with it working on the [heroku-migration-api-upgrade branch](https://circleci.com/workflow-run/85313d0f-c9d9-437b-b384-3264fb36ce6d).

For those reviewing, please sanity check https://github.com/Financial-Times/next-syndication-api/pull/104/commits/c9885a563621002e4db7c51fb40436d23165c5dc (adds `/__gtg` for downloads app, which simplifies gtg and smoke tests) and https://github.com/Financial-Times/next-syndication-api/pull/104/commits/dfad7e7959fd40cff59f556ba01448198fb9ac57 (removes the seemingly redundant `/syndication/__gtg`).

File: server/app.js [hunk 1]
```diff
@@ -57,8 +57,6 @@ const middleware = [
 
 app.get('/__gtg', (req, res) => res.sendStatus(200));
 
-app.get('/syndication/__gtg', (req, res) => res.sendStatus(200));
```
Comment (leggsimon) [inline server/app.js]: it’s possible some healthchecks depend on this but we can probably wait until it’s merged to find out

Comment (taktran) [reply server/app.js]: Health checks seem ok

File: Makefile [hunk 2]
```diff
@@ -4,9 +4,9 @@ node_modules/@financial-times/n-gage/index.mk:
 
 -include node_modules/@financial-times/n-gage/index.mk
 
-APP_NAME := ft-next-syndication-api
-
-TEST_APP := "${APP_NAME}-${CIRCLE_BUILD_NUM}"
+VAULT_NAME=ft-next-syndication-api
```
[the remark this answers is not in the document]
Comment (taktran) [reply Makefile]: Added a card to look into it: https://github.com/Financial-Times/next/projects/2#card-16504195

**Pull request merged**